.travis.yml

```diff
@@ -1,28 +1,87 @@
 language: php
 
-php:
-    - 7.0
-    - 7.1
-    - 7.2
-
 cache:
-    directories:
-        - $HOME/.composer/cache/files
+  directories:
+    - $HOME/.composer/cache/files
 
-before_install:
-    - phpenv config-rm xdebug.ini
-    - composer validate
+stages:
+  - stan
+  - test
 
-install:
-    - composer update --prefer-dist --prefer-stable --no-interaction
+jobs:
+  include:
+    - stage: Stan
+
+      php: 7.2
+
+      install:
+        - composer install
+
+      script:
+        - vendor/bin/phpstan analyse -l max -c phpstan.neon src tests
+
+    - &TEST
```

Comment:
This creates an anchor, which we can later reference and extend from. 

Also see https://blog.daemonl.com/2016/02/yaml.html.